config.json

```diff
@@ -96,6 +96,11 @@
       "topics": []
     },
     {
+      "slug": "collatz-conjecture",
+      "difficulty": 4,
```

Comment:
The algebra necessary for this problem is simple, but I bumped the difficulty up a bit since most solutions will require recursion or some equivalent construct. This is rated less difficult than flatten-array, another exercise that lends itself to a recursive implementation but that is notably more difficult conceptually (currently rated a 5; probably too low?)